### PR TITLE
Push frontend and backend

### PR DESCRIPTION
**Pull request by RahulGopathi**
### Description

- Pushed the frontend and backend required for the ACAC portal
- Changed `SOCIETIES` text to `BOARDS` in frontend

   alt text

- Removed `people` section from the portal

  alt text
 
- Updated `README.md` file of Backend

### Type of Change:

- Code
- User Interface
- Documentation

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have commented my code or provided relevant documentation
- [ ] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

### CODE REVIEW

File: acac_frontend/.graphqlconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Gymkhana GraphQL Schema",
+  "schemaPath": "./src/plugins/schema.json",
+  "extensions": {
+    "endpoints": {
+      "Gymkhana Dev GraphQL Endpoint": {
+        "url": "http://localhost:8000/graphql",
```
Comment (DarshitJain04) [inline acac_frontend/.graphqlconfig:7]: For now we can move forward with this. Afterwards make the base url dynamic (using env variables)

File: acac_portal/Pipfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+django = "==2.2.13"
```
Comment (DarshitJain04) [inline acac_portal/Pipfile:7]: Is this version compatible with the DRF version?
Maybe you guys can shift to a newer version of Django

File: acac_frontend/.graphqlconfig (hunk 1 above)
Comment (RahulGopathi) [reply acac_frontend/.graphqlconfig:7]: yeah

File: acac_portal/Pipfile (hunk 2 above)
Comment (RahulGopathi) [reply acac_portal/Pipfile:7]: Yes, it is compatible with DRF but yeah we can shift to Django 3 I believe

Comment (DarshitJain04) [reply acac_portal/Pipfile:7]: Shift to a newer version and push the changes

Comment (RahulGopathi) [reply acac_portal/Pipfile:7]: ok

**Review by DarshitJain04 — APPROVED** [review]

**Pull request merged**